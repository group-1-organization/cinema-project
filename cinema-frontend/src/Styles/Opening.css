.hours {
    text-align: right;
}

.col-m-6 {
    width: 50%;
    float: left;
    font-size: large;
}

.col-mr-6 {
    width: 40%;
    float: right;
    font-size: large;
}

.col-m-12 {
    width: 20%;
}

.op-font-colour {
    color: white;
}