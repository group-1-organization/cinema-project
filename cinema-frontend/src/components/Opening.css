.header {
    padding: 50px;
    text-align: center;
    background: #0e254d;
    color: white;
    font-size: 30px;
}

.hours {
    text-align: right;
}

.col-m-6 {
    width: 50%;
    float: left;
    font-size: large;
}

.col-mr-6 {
    width: 40%;
    float: right;
    font-size: large;
}

.col-m-12 {
    width: 20%;
}

.op-font-colour {
    color: white;
}