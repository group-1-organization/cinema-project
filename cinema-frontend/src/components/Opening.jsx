import { Container } from 'react-bootstrap';
import '../Styles/Opening.css';
import '../App.css'

const Opening = () => {
    return (
        <div>
            <div class="page-header">
                <h1>Opening Hours</h1>
            </div>
            <br />
            <Container>
                <center class="op-font-colour">
                    <div class="col-m-12">
                        <p class="col-m-6">Monday: </p>
                        <p class="col-mr-6 hours">9:00-23:00</p>
                    </div>
                    <div class="col-m-12">
                        <p class="col-m-6">Tuesday:</p>
                        <p class="col-mr-6 hours">9:00-23:00</p>
                    </div>
                    <div class="col-m-12">
                        <p class="col-m-6">Wednesday:</p>
                        <p class="col-mr-6 hours">9:00-23:00</p>
                    </div>
                    <div class="col-m-12">
                        <p class="col-m-6">Thursday:</p>
                        <p class="col-mr-6 hours">9:00-23:00</p>
                    </div>
                    <div class="col-m-12">
                        <p class="col-m-6">Friday:</p>
                        <p class="col-mr-6 hours">9:00-23:00</p>
                    </div>
                    <div class="col-m-12">
                        <p class="col-m-6">Saturday:</p>
                        <p class="col-mr-6 hours">9:00-23:00</p>
                    </div>
                    <div class="col-m-12">
                        <p class="col-m-6">Sunday:</p>
                        <p class="col-mr-6 hours">9:00-23:00</p>
                    </div>
                </center>
            </Container>
        </div>
    );
}

export default Opening;