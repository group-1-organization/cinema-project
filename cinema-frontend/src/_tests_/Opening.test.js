import renderer, { act } from 'react-test-renderer';
import Opening from '../components/Opening';

test('Opening page matches snapshot', () => {
    const tree = renderer.create(<Opening />);

    expect(tree).toMatchSnapshot();
});