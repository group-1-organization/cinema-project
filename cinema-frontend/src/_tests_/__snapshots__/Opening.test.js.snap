// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Opening page matches snapshot 1`] = `
<div>
  <div
    class="page-header"
  >
    <h1>
      Opening Hours
    </h1>
  </div>
  <br />
  <div
    className="container"
  >
    <table
      className="table-format table table-borderless"
    >
      <tbody>
        <tr>
          <td>
            Monday:
          </td>
          <td>
            11:00 - 23:00
          </td>
        </tr>
        <tr>
          <td>
            Tuesday:
          </td>
          <td>
            11:00 - 23:00
          </td>
        </tr>
        <tr>
          <td>
            Wednesday:
          </td>
          <td>
            11:00 - 23:00
          </td>
        </tr>
        <tr>
          <td>
            Thursday:
          </td>
          <td>
            11:00 - 23:00
          </td>
        </tr>
        <tr>
          <td>
            Friday:
          </td>
          <td>
            11:00 - 23:00
          </td>
        </tr>
        <tr>
          <td>
            Saturday:
          </td>
          <td>
            9:00 - 23:00
          </td>
        </tr>
        <tr>
          <td
            className={true}
          >
            Sunday:
          </td>
          <td>
            9:00 - 23:00
          </td>
        </tr>
      </tbody>
    </table>
  </div>
</div>
`;
